foundation-extensions/profile/profile-api-references.md

```diff
@@ -83,6 +83,15 @@ public static func updateUserAttributes(attributeDict: [String: Any])
 
 You want to update `username, usertype` of a user obtained in the log in page :
 
+**Swift**
```

Comment:
Let's also include the extensionVersion and registerExtension APIs to this page.